Eatery/Extensions/UIColor+ColorScheme.swift

```diff
@@ -26,6 +26,9 @@ extension UIColor {
 
     /// 0x4A90E2
     static let eateryBlue = colorFromCode(0x4A90E2)
+    
+    /// 0xb7d3f3
```

Comment:
What's the point of having the hex in a comment here (and in the rest of this file)?